security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,44 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-16-0_{context}"]
+== {cert-manager-operator} 1.16.0
+
+Issued: 2025-05-27
+
+The following advisories are available for the {cert-manager-operator} 1.16.0:
+
+* link:https://access.redhat.com/errata/RHEA-2025:8163[RHEA-2025:8163]
+* link:https://access.redhat.com/errata/RHEA-2025:8164[RHEA-2025:8164]
+* link:https://access.redhat.com/errata/RHEA-2025:8165[RHEA-2025:8165]
+* link:https://access.redhat.com/errata/RHEA-2025:8198[RHEA-2025:8198]
+
+Version `1.16.0` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.16.4`. For more information, see the link:https://cert-manager.io/docs/releases/release-notes/release-notes-1.16/#v1164[cert-manager project release notes for v1.16.4].
+
+[id="cert-manager-operator-1-16-0-features-enhancements_{context}"]
+=== New features and enhancements
+
+*Disconnected installation support*
```

Comment:
May be it'd be more accurate to use:
```suggestion
*Disconnected environment support*
```

Because apart from the operator installation, its functionality has also been validated.